packages/@aws-cdk/aws-cognito/test/user-pool-client.test.ts

```diff
@@ -42,6 +42,37 @@ describe('User Pool Client', () => {
     expect(() => client2.userPoolClientName).toThrow(/available only if specified on the UserPoolClient during initialization/);
   });
 
+  // eslint-disable-next-line jest/no-focused-tests
+  test.only('client with secret', ()=>{
```

Comment:
Why are you using `test.only` here? This means that no other tests in this package will run. We very much do not want that.
```suggestion
  test('client with secret', () => {
```

Reply:
That was not intentional. Leftover from local testing. Corrected.